controllers/v1/observationSubmissionsController.js

```diff
@@ -139,7 +146,7 @@ module.exports = class ObservationSubmissions extends Abstract {
           entityDocument.metaInformation.registryDetails = entityDocument.registryDetails;
         }
 
-        let solutionDocument = await solutionsHelper.solutionDocuments(
+        let solutionDocument = await solutionQueries.solutionDocuments(
```

Comment:
@borkarsaish65 @borkarsaish65 query call should not happen from controller

Reply:
done